grakn-test-tools/src/main/java/ai/grakn/util/GraknTestSetup.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.util;
+
+
+/**
+ * <p>
+ *     Helpes Setup Grakn Test Environment
+ * </p>
+ *
+ * <p>
+ *     Contains utility methods and statically initialized environment variables to control
+ *     Grakn unit tests.
+ * </p>
+ *
+ * @author borislav
+ *
+ */
+public class GraknTestSetup {
+    private static String CONFIG = System.getProperty("grakn.test-profile");
+
+    /**
+     * Starts cassandra if needed.
+     */
+    public static void ensureCassandraRunning() {
+        if (GraknTestSetup.usingTitan()) {
+            EmbeddedCassandra.start();
+        }
```

Comment:
I'm not 100% sure if we should call this `ensureCassandraRunning`, since it only sometimes does that. Maybe something to indicate it initialises the graph backend? Like `setupGraphDependencies` or similar.